mod/caching/set/type_plus_right/wikirate_analysis/metric.rb

```diff
@@ -0,0 +1,37 @@
+include Card::CachedCount
+
+expired_cached_count_cards do |changed_card|
+  # "right_plus":[["topic",{"refer_to":"_2"}],
+  #  ["_ll",{"right_plus":["*cached count",{"ne":0}]}] ]
+
+  unless changed_card.simple? || (changed_card.right_id != CachedCountID &&
+                                  changed_card.type_id != PointerID)
```

Comment:
You can use `next` in a block as guard clause. That would eliminate the codeclimate warning.